napari/layers/image/image.py

```diff
@@ -105,12 +113,21 @@ def __init__(
         clim=None,
         clim_range=None,
         interpolation='nearest',
+        opacity=1,
```

Comment:
could we make these keyword-only arguments by adding the `*` as a separate argument?

Reply:
we already have a `*` at line 109. why do we need another?

```python
    def __init__(
        self,
        image,
        *,
        metadata=None,
        multichannel=None,
        colormap='gray',
        clim=None,
        clim_range=None,
        interpolation='nearest',
        opacity=1,
        blending='translucent',
        visible=True,
        name=None,
        **kwargs,
    ):
```

Reply:
didn't see that, mb